testdata/templates/custom_actions.tpl

```diff
@@ -180,7 +180,6 @@
     "account": {
       "customActionsTriggerDefinition": {
         "extendedTeamAccess": {
-          "nodes": [ {{ template "team_1" }} ],
```

Comment:
why was this removed?  Don't we just need to remove "tags" not teams?

Reply:
This was part of some crude troubleshooting and should be put back in